samples/tfm_integration/psa_crypto/Kconfig

```diff
@@ -19,6 +19,11 @@ config PSA_SHELL
 	help
 	  Enabling this option will make the 'psa' shell command available.
 
+config PSA_IMPORT_KEY
+	bool "Enable support for importing private key data"
+	help
+	  Enable support for importing private key data using PSA APIs.
+
 choice
 	prompt "Private Key"
 	default PRIVATE_KEY_RANDOM
```

Comment:
Does this choice depend on `PSA_IMPORT_KEY` being set to true to be relevant? If so `depends on` should be added.

Reply:
Yes, fixed.